internal/cmd/templates/templates.go

```diff
@@ -52,21 +58,49 @@ func NewTemplateManager() (*TemplateManager, error) {
 		minimalTemplate:  minimalTmpl,
 		protocolTemplate: protocolTmpl,
 		browserTemplate:  browserTmpl,
+		fs:               fs,
 	}, nil
 }
 
 // GetTemplate selects the appropriate template based on the type
-func (tm *TemplateManager) GetTemplate(templateType string) (*template.Template, error) {
-	switch templateType {
+func (tm *TemplateManager) GetTemplate(tpl string) (*template.Template, error) {
+	// First check built-in templates
+	switch tpl {
 	case MinimalTemplate:
 		return tm.minimalTemplate, nil
 	case ProtocolTemplate:
 		return tm.protocolTemplate, nil
 	case BrowserTemplate:
 		return tm.browserTemplate, nil
-	default:
-		return nil, fmt.Errorf("invalid template type: %s", templateType)
 	}
+
+	// Then check if it's a file path
```

Comment:
Non-blocking, feel free to ignore.

I don't think it's absolutely worth it for this specific feature, but more as a best practice, it could be worth cleaning up and transforming the path to an absolute path here, at least as we work with it internally: `tplPath, err := filepath.Abs(filepath.Clean(tpl))`.